src/jit/lowerxarch.cpp

```diff
@@ -4573,6 +4579,32 @@ void Lowering::SetMulOpCounts(GenTreePtr tree)
 }
 
 //------------------------------------------------------------------------------
+// SetContainsAVXFlags: default value of isFloatingType is true, we set the
+// ContainsAVX flag when floating type value is true, when SIMD vector size is
+// 32 bytes, it is 256bit AVX instruction and we set Contains256bitAVX flag too
+//
+// Arguments:
+//    isFloatingType    - is floating type
+//    sizeOfSIMDVector  - SIMD Vector size
+//
+void Lowering::SetContainsAVXFlags(bool isFloatingType, unsigned sizeOfSIMDVector)
```

Comment:
Suggestion: We can indicate the default values of params as follows:

void Lowering::SetContainsAVXFlags(bool isFloatingType  /* = true */, 
                                                                 unsigned sizeOfSIMDVector /* = 0 */)